test/datadog/lambdacontext.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+# This is an example of the lambda context provided to a Ruby-runtimed lambda
+# Use dot-notation to access these properties
+class LambdaContext
```

Comment:
This is a really useful mock, I just ran into a similar issue with the context with the native tracing stuff.